src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -75,6 +75,8 @@ public class BuildApk : Task
 
 		public string Debug { get; set; }
 
+		public bool UseRuntimeWithDebugSymbols { get; set; }
```

Comment:
This isn't (nor should be!) limited to just "runtime" libraries; this should apply to *all* native libraries, and *does*, because it's in `AddNativeLibrary()`. The name should thus imply that.

I would suggest `PreferUnstrippedNativeLibraries` or `PreferNativeLibrariesWithDebugSymbols` or something.